ehsm_kms_service/test/security_test/createkey.py

```diff
@@ -0,0 +1,197 @@
+import requests
+import json
+import argparse
+import base64
+import time
+import random
+import hmac
+from hashlib import sha256
+from collections import OrderedDict
+import urllib.parse
+import pytest
+from conftest import test_url
+
+import _utils_
+
+#  Max timestamp diff is 10 mins set in eHSM
+#  Thus the nonce is set to twice timestamp diff
+#  MAX_TIME_STAMP_DIFF = 10 * 60 * 1000
+#  NONCE_CACHE_TIME = MAX_TIME_STAMP_DIFF * 2
+NONCE_CACHE_TIME_MINS = 20
+
+
+def init_params(appid, appkey, timestamp_id, keyspec, origin, sign=None):
+    payload = OrderedDict()
```

Comment:
please replace the "appkey" to "apikey" in the whole file.

Reply:
fixed